tensorflow/g3doc/api_docs/python/script_ops.md

```diff
@@ -17,7 +17,7 @@ TensorFlow operators.
 
 ### `tf.py_func(func, inp, Tout, stateful=True, name=None)` {#py_func}
 
-Wraps a python function and uses it as a tensorflow op.
+Wraps a python function and uses it as a TensorFlow op.
```

Comment:
and this one, and python/train.md.

All the other are changes seem valid to me, so thanks!